### ISSUE TITLE
[Bug] Any chunks with an emissive block texture loaded in them visually break and don't display any textures

### ISSUE THREAD

**Issue by C4sDaBomb** [sp614x/optifine#6549]
## Description of Issue
When (most) chunks contain a block with an emissive overlay texture (using the OptiFine emissive textures feature), blocks such as dirt, grass, most plants, and sand glitch visually, become textureless, and look like a solid cube having a solid color pertaining to the block's real texture. An x-ray effect is also observed, with mild strobing that may be dangerous to epileptics. The glitch only affects chunks where the block that is emissive was present prior to loading the resource pack, for the most part. It could also be only affecting of naturally generated blocks with the emissive overlay.

## Steps to Reproduce
1. Create a resource pack with a single block texture (preferrably one that naturally generates often; I used dirt in my example).
2. Create an emissives.properties file and set it up to enable emissive textures.
3. Create an emissive overlay of the block in the resource pack.
4. Load the resource pack.
5. The glitch should activate.

## OptiFine Version
1.18.1 OptiFine HD U H5 pre4

The glitch only started happening around the early 1.18 versions of OptiFine and has persisted since then.

## Installation Method
I installed it in the way I always have, standalone. I opened the jar with Java Platform SE Binary and followed the prompt instructions. 

## Prior Testing

Does this happen in Vanilla without OptiFine?

No, it is due to an OptiFine-only feature.

## Additional Information
I made a bug report in the #bug-hunting channel on the discord server. My discord is C4#6227 if you want to view it. I found two other users with the same issue, which I have mentioned in the report.

**Comment by KaiAF** [comment]
It looks like anisotropic filtering actually causes the issue, maybe a bit of both. You can try to "fix" it by disabling anisotropic filtering.

**Comment by C4sDaBomb** [comment]
> It looks like anisotropic filtering actually causes the issue, maybe a bit of both. You can try to "fix" it by disabling anisotropic filtering.

Yeah for some reason turning it off fixes the problem. Could be an incompatibility
